test/presenters/statistics_announcement_presenter_test.rb

```diff
@@ -0,0 +1,128 @@
+
+  test 'presents from as links to organisations' do
+    item = presented_statistics_announcement({
+      "links" => {
+        "organisations" => [{ "title" => 'Org', "base_path" => '/orgs/org'}]
+      }
+    })
+    links = [
+      link_to('Org', '/orgs/org')
+    ]
+    assert_equal links, item.from
+  end
+
+  test 'presents part_of as links to a topic' do
+    item = presented_statistics_announcement({
+      "links" => {
+        "policy_areas" => [{ "title" => 'Policy Area', "base_path" => '/orgs/policy-area'}]
+      }
+    })
+    links = [
+      link_to('Policy Area', '/orgs/policy-area')
+    ]
+    assert_equal links, item.part_of
+  end
+
+  test 'presents release_date' do
+    item = presented_statistics_announcement({
+      "details" => {
+        "display_date" => "April to May 2016"
+      }
+    })
+    assert_equal 'April to May 2016', item.release_date
+  end
+
+  test 'presents release_date_and_status when confirmed' do
+    item = presented_statistics_announcement({
+      "details" => {
+        "display_date" => "April to May 2016",
+        "state" => "confirmed"
+      }
+    })
+    assert_equal 'April to May 2016 (confirmed)', item.release_date_and_status
+  end
+
+  test 'presents release_date_and_status when provisional' do
+    item = presented_statistics_announcement({
+      "details" => {
+        "display_date" => "April to May 2016",
+        "state" => "provisional"
+      }
+    })
+    assert_equal 'April to May 2016 (provisional)', item.release_date_and_status
+  end
+
+  test 'presents release_date_and_status when cancelled' do
+    item = presented_cancelled_statistics_announcement
+    assert_equal 'April to May 2016', item.release_date_and_status
+  end
+
+  test "present other metadata when confirmed" do
+    item = presented_statistics_announcement({
+      "details" => {
+        "display_date" => "April to May 2016",
+        "state" => "confirmed"
+      }
+    })
+    other = {
+      "Release date" => "April to May 2016 (confirmed)"
+    }
+    assert_equal other, item.other_metadata
+  end
+
+  test "present other metadata when cancelled" do
+    item = presented_cancelled_statistics_announcement
+    other = {
+      "Proposed release" => "April to May 2016",
+      "Cancellation date" => "17 January 2016 2:19pm"
+    }
+    assert_equal other, item.other_metadata
+  end
+
+  test "shows the cancellation reason when cancelled" do
+    item = presented_cancelled_statistics_announcement
+
+    assert_equal "Release cancelled for operational reasons.", item.cancellation_reason
+  end
+
+  test 'knows if an item is a national statistic' do
+    item = presented_statistics_announcement({
+      "details" => {
+        "format_sub_type" => "official"
+      }
+    })
+    assert_not item.national_statistics?
+
+    item = presented_statistics_announcement({
+      "details" => {
+        "format_sub_type" => "national"
+      }
+    })
+    assert item.national_statistics?
+  end
+
+private
```

Comment:
Thanks, addressed now. Opening a separate PR to remove them in other format's tests.